frame/support/src/storage/mod.rs

```diff
@@ -321,10 +321,18 @@ pub trait IterableStorageMap<K: FullEncode, V: FullCodec>: StorageMap<K, V> {
 	/// this, you'll get undefined results.
 	fn iter() -> Self::Iterator;
 
+	/// Enumerate all elements in the map after a specified `starting_raw_key` in no particular
+	/// order. If you alter the map while doing this, you'll get undefined results.
```

Comment:
I still think we need to say what is the order. If there is no specified order, how can people do some partial iteration and resume them.